fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/RecordQueryPlanner.java

```diff
@@ -1195,7 +1236,7 @@ private ScoredPlan planOrderedUnion(@Nonnull PlanContext planContext, @Nonnull L
         if (unionPlan.getComplexity() > complexityThreshold) {
             throw new RecordQueryPlanComplexityException(unionPlan);
         }
-        return new ScoredPlan(1, unionPlan, Collections.emptyList(), anyDuplicates, includedRankComparisons);
+        return new ScoredPlan(0, unionPlan, Collections.emptyList(), anyDuplicates, includedRankComparisons);
```

Comment:
I'm not sure whether to gate this behind the config parameter, too. That feels hacky, but is probably safer.

Reply:
Yeah, that is probably safer, since reasoning about these constant scores is sometimes tricky.